tests/functional/test_rsid_coordinates_resolver.py

```diff
@@ -0,0 +1,193 @@
+    assembly = "GRCh37"
+    (chromosome, start, end) = get_ensemblid_coordinates(id, assembly)
+    assert chromosome == "chr7"
+    assert start == 140419127
+    assert end == 140624729
+
+
+def test_get_ensemblid_coordinates_ncbi36():
+    id = "ENSG00000157764"
+    assembly = "NCBI36"
+    (chromosome, start, end) = get_ensemblid_coordinates(id, assembly)
+    assert chromosome == ""
+    assert start == ""
+    assert end == ""
+
+
+def test_get_ensemblid_coordinates_GRCm39():
+    id = "ENSMUSG00000031201"
+    assembly = "GRCm39"
+    (chromosome, start, end) = get_ensemblid_coordinates(id, assembly)
+    assert chromosome == "chrX"
+    assert start == "74460234"
+    assert end == "74497607"
+
+
+def test_get_ensemblid_coordinates_GRCm38():
+    id = "ENSMUSG00000031201"
+    assembly = "GRCm38"
+    (chromosome, start, end) = get_ensemblid_coordinates(id, assembly)
+    assert chromosome == "chrX"
+    assert start == 75416628
+    assert end == 75454001
+
+
+def test_get_ensemblid_coordinates_GRCm37():
+    id = "ENSMUSG00000031201"
+    assembly = "GRCm37"
+    (chromosome, start, end) = get_ensemblid_coordinates(id, assembly)
+    assert chromosome == "chrX"
+    assert start == 72661967
+    assert end == 72699340
+
+
+def test_get_ensemblid_coordinates_error():
+    id = "ENUSG00000157764"
+    assembly = "GRCh38"
+    (chromosome, start, end) = get_ensemblid_coordinates(id, assembly)
+    assert chromosome == ""
+    assert start == ""
+    assert end == ""
+
+
+def test_get_rsid_coordinates_from_ensembl_grch38():
+    assembly = "GRCh38"
+    rsid = "rs56116432"
+    chromosome, start, end = get_rsid_coordinates_from_ensembl(assembly, rsid)
+    assert chromosome == "chr9"
+    assert start == 133256041
+    assert end == 133256042
+
+
+def test_get_rsid_coordinates_from_ensembl_grch37():
+    assembly = "GRCh37"
+    rsid = "rs56116432"
+    chrom, start, end = get_rsid_coordinates_from_ensembl(assembly, rsid)
+    assert chrom == "chr9"
+    assert start == 136131428
+    assert end == 136131429
+
+
+def test_get_rsid_coordinates_no_atlas():
+    assembly = "GRCh38"
+    rsid = "rs56116432"
+    chrom, start, end = get_rsid_coordinates(rsid, assembly)
+    assert chrom == "chr9"
+    assert start == 133256041
+    assert end == 133256042
+
+
+def test_get_coordinates():
+    query_term = "chr10:5894499-5894500"
+    chrom, start, end = get_coordinates(query_term)
+    assert chrom == "chr10"
+    assert start == 5894499
+    assert end == 5894500
+
+
+def test_get_coordinates_rsid():
+    query_term = "rs56116432"
+    chrom, start, end = get_coordinates(query_term)
+    assert chrom == "chr9"
+    assert start == 136131428
+    assert end == 136131429
+
+
+def test_get_coordinates_ensemblid():
+    query_term = "ENSG00000157764"
+    assembly = "GRCh38"
+    chrom, start, end = get_coordinates(query_term, assembly)
+    assert chrom == "chr7"
+    assert start == 140719327
+    assert end == 140924929
+
```

Comment:
Maybe could use https://docs.pytest.org/en/6.2.x/parametrize.html to avoid repeating. Not sure if this is just a matter of opinion though.

Reply:
maybe better to keep it like it is, to keep it clear which test fails.